content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -33,79 +33,88 @@ further_reading:
+</div>
 
-## Automatic Instrumentation
+If you don't use the in-app documentation to guide you, you can follow these instructions to begin tracing .NET applications: 
 
-Automatic instrumentation can collect performance data about your application with zero code changes and minimal configuration. The .NET Tracer automatically instruments all [supported libraries][1] out of the box.
+#### IIS applications 
 
-Automatic instrumentation captures:
+To start tracing an IIS application:
 
-- Execution time of instrumented calls
-- Relevant trace data, such as URL and status response codes for web requests or SQL queries for database access
-- Unhandled exceptions, including stacktraces if available
-- A total count of traces (e.g. web requests) flowing through the system
+1. Install and configure the [Windows Datadog Agent][2]. 
 
-### Installation
+2. Download the .NET Tracer [MSI installer][3].
 
-To use automatic instrumentation on Windows, install the .NET Tracer on the host by running the [MSI installer for Windows][4] as administrator. Choose the installer for the architecture that matches the operating system (x64 or x86).
+3. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
 
-After installing the .NET Tracer, restart applications so they can read the new environment variables. To restart IIS, run the following commands as administrator:
+4. Install the .NET Tracer MSI installer with administrator privileges.
 
-```cmd
-net stop /y was
-net start w3svc
-```
+5. Restart IIS using the following command as an administrator: 
+
+   ```text
+   net stop /y was
+   net start w3svc
+   ```
+6. Create application load. 
 
-### Required Environment Variables
+7. Visit [APM Live Traces][4]. 
 
-If your application runs in IIS, skip the rest of this section.
+#### Non-IIS applications
 
-For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
+To enable automatic instrumentation on non-IIS Windows applications, you must set two environment variables before starting your application.
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+<div class="alert alert-warning"> 
+  <strong>Note:</strong> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. Don't set these environment variables globally because this causes all .NET processes on the host to load the profiler.
+</div>
+
+Set these environment variables for the non-IIS applications you want to instrument:
```

Comment:
You should either keep this line or the line before the alert box, as they say the same thing